controller/token.go

```diff
@@ -126,6 +126,26 @@ func (c *TokenController) Generate(ctx *app.GenerateTokenContext) error {
 			ProviderType:          account.KeycloakIDP,
 			RegistrationCompleted: true,
 		}
+
+		err = transaction.Transactional(c.app, func(tr transaction.TransactionalResources) error {
```

Comment:
how does this apply to `dev user in developer mode` only (as the title of the PR suggests)?

Reply:
This is the code which makes sure that you have developer mode enabled where we are looking only for `developer` user https://github.com/fabric8-services/fabric8-auth/blob/c5d5e91e3729356ac779f01184c0353b9ddd8ef6/controller/token.go#L99-L113

Reply:
ah of course, I missed that :/ Thanks for the clarification ;)